tests/manage/pv_services/test_create_pvc_with_worker_node_down.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import random
+import pytest
+
+from ocs_ci.ocs.resources import pod
+
+from ocs_ci.ocs import node
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, tier4, tier4a
+from tests.helpers import get_worker_nodes
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+class TestCreatePvcWithWorkerNodeDown(ManageTest):
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
+        """
+        Make sure all nodes are up again
+
+        """
+        def finalizer():
+            nodes.restart_nodes_teardown()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.polarion_id("OCS-1628")
+    def test_create_delete_pvc_parallel(self, project_factory, storageclass_factory, dc_pod_factory, nodes):
+        """
+        PV provisioning with one worker down
+
+        """
+
+        # Get worker nodes
+        worker_node_list = get_worker_nodes()
+        log.info(f"Current available worker nodes are {worker_node_list}")
+
+        mgr_pod_obj = pod.get_mgr_pods()
+        log.info(mgr_pod_obj[0].name)
+        mgr_node_name = pod.get_pod_node(mgr_pod_obj[0]).name
+        worker_node_list.remove(mgr_node_name)
+        selected_worker_node = random.choice(worker_node_list)
+        log.info(f"Stopping Node {selected_worker_node}")
+        selected_worker_node_obj = node.get_node_objs(selected_worker_node)
+        log.info(selected_worker_node_obj[0].name)
+        nodes.stop_nodes(selected_worker_node_obj)
+        toolbox_pod_obj = pod.get_ceph_tools_pod()
+        toolbox_node_name = pod.get_pod_node(toolbox_pod_obj).name
+        if selected_worker_node == toolbox_node_name:
+            helpers.wait_for_ct_pod_recovery()
+        project_obj = project_factory()
+        rbd_sc_obj = helpers.default_storage_class(interface_type=constants.CEPHBLOCKPOOL)
+        cephfs_sc_obj = helpers.default_storage_class(interface_type=constants.CEPHFILESYSTEM)
+        all_pvc_obj, dc_pod_obj = list()
+        log.info("Creating pvc in parallel")
+        rbd_pvcs_obj = helpers.create_multiple_pvc_parallel(
+            sc_obj=rbd_sc_obj, namespace=project_obj.namespace, number_of_pvc=25, size="10Gi",
+            access_modes=[constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX]
```

Comment:
Will this create rbd block type volume to support rwx ?

Reply:
yes